src/euphorie/client/browser/risk.py

```diff
@@ -634,7 +672,17 @@ def skip_evaluation(self):
             return True
         return False
 
+    @memoize
     def get_existing_measures(self):
+        return [
```

Comment:
isn't the code in `get_existing_measures` equivalent to:
```
return list(self.context.in_place_standard_measures) + list(self.context.in_place_custom_measures)
```
?

Reply:
Yes, that is actually much simpler and easier to read, thanks